internal/asts/parse.go

```diff
@@ -3,19 +3,13 @@ package asts
 import (
 	"fmt"
 	"go/ast"
-	"go/build"
 	"go/parser"
 	"go/token"
 )
 
 func ParsePkgPath(pkgPath string) (*ast.Package, error) {
-	// resolve pkgPath as a vendor if possible
-	tpkg, err := SrcImporter.Import(pkgPath)
-	if err != nil {
-		return nil, err
-	}
 	// resolve directory of pkg path
-	bpkg, err := bImport(tpkg.Path(), ".", build.FindOnly)
+	bpkg, err := bImport(pkgPath, ".", 0)
```

Comment:
why not `build.FindOnly` ?

Reply:
To get a correct name in `bpkg.Name`. If you use `build.FindOnly` then `bpkg.Name` is an empty string.

Reply:
you're great!